spec/system/discussion_spec.rb

```diff
@@ -0,0 +1,89 @@
+require "rails_helper"
+require_relative "../modules/helpers"
+require_relative "../matchers/category_matchers"
+require_relative "../matchers/discussion_matchers"
+require_relative "../matchers/shared_matchers"
+
+RSpec.describe "Discussion,", type: :system do
+  include SharedMatchers
+  include SessionHelpers
+  include CategoryHelpers
+  include DiscussionHelpers
+
+  context "when logged in as admin," do
+    before(:each) do
+      login_as("admin")
+    end
+
+    let(:category_name) { "Test Category Name" }
+    let(:discussion_title) { "Test Discussion Title" }
+    let(:discussion_body) { "Test discussion body" }
+
+    it "can be created" do
+      create_discussion(discussion_title, discussion_body, Category.create(name: category_name, position: 1, description: CATEGORY_DESCRIPTION))
```

Comment:
You've got `let` calls for everything but description.  For consistency's sake you should do the same with `CATEGORY_DESCRIPTION`.